src/game.cpp

```diff
@@ -4291,29 +4215,23 @@ void ingameHud()
 						}
 					}
 				}
-				*inputPressedForPlayer(player, impulses[IN_DEFEND]) = 0;
-				inputs.controllerClearInput(player, INJOY_GAME_DEFEND);
+				input.consumeBinaryToggle("Cast Spell");
+				input.consumeBinaryToggle("Block");
 			}
 		}
 		players[player]->magic.resetQuickCastSpell();
 
-		if ( !command && *inputPressedForPlayer(player, impulses[IN_TOGGLECHATLOG])
-			|| (players[player]->shootmode && inputs.bControllerInputPressed(player, INJOY_GAME_TOGGLECHATLOG)) )
+		if ( !command && input.consumeBinaryToggle("Open Log") )
 		{
-			hide_statusbar = !hide_statusbar;
```

Comment:
Does the chatlog still get opened from the old config file? Can't turn it off anymore with this